modules/identity/src/Volo.Abp.Identity.Blazor/AbpIdentityBlazorMessageLocalizerExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Blazorise;
+using Microsoft.Extensions.Localization;
+
+namespace Volo.Abp.Identity.Blazor
+{
+    public static class AbpIdentityBlazorMessageLocalizerExtensions
+    {
+        public static string Localize<T>(this IStringLocalizer<T> stringLocalizer, string format, IEnumerable<string> arguments)
+        {
+            try
+            {
+                return arguments?.Count() > 0
+                    ? string.Format(stringLocalizer[format], LocalizeMessageArguments(stringLocalizer, arguments)?.ToArray())
+                    : stringLocalizer[format];
```

Comment:
I have now used `stringLocalizer[format, arguments]`. And the reason for all of this code is because I need to localize arguments along with messages because they have "fallback" logic.

```
return stringLocalizer[$"DisplayName:{argument}"]
  ?? stringLocalizer[argument]
  ?? argument;
```